src/main/python/index.ts

```diff
@@ -83,7 +84,16 @@ export async function pipxEnsurepath(): Promise<void> {
 
 export async function installPoetry(): Promise<void> {
   const py = process.env.PY_INTERPRETER ?? "python";
-  await execCommand(new Command(`${py} -m pipx install poetry --force`));
+  const localDir = `${os.homedir()}/.local`;
```

Comment:
I think it is better to replace with `path.join(os.homedir(), '.local');`?

Reply:
Yeah, sounds good. Modified as requested. Thanks